src/AppBundle/Controller/HomeController.php

```diff
@@ -29,26 +36,47 @@ public function indexAction()
     public function searchAction(Request $request)
     {
         $searchInput = $request->query->get('tag');
-        $result = $this->getDoctrine()->getRepository('AppBundle:Category')->searchByTitle($searchInput);
+        $polls = $this->getDoctrine()->getRepository('AppBundle:Poll')->searchByCategoryOrPollName($searchInput);
+        if (!$polls) {
+            return $this->render('AppBundle:Home:index.html.twig', [
+                'polls' => $polls,
+                'error' => 'Sorry! No entries were found :('
+            ]);
+        }
+        return $this->render('AppBundle:Home:index.html.twig', [
+            'polls' => $polls
+        ]);
+    }
 
-        return new JsonResponse(array_map(function (Category $category) {
+    /**
+     * @Route("/searchAutoComplete", name="searchAutoComplete")
+     */
+    public function searchAutoCompleteAction(Request $request)
+    {
+        $searchInput = $request->query->get('tag');
+        $tags = $this->getDoctrine()->getRepository('AppBundle:Category')->searchByTitle($searchInput);
+        $polls = $this->getDoctrine()->getRepository('AppBundle:Poll')->searchByTitle($searchInput);
+        $tagsArray = array_map(function (Category $category) {
             return $category->getTitle();
-        }, $result
-        ));
+        }, $tags);
+        $pollsArray = array_map(function (Array $poll) {
```

Comment:
`array` pagal psr-2 http://www.php-fig.org/psr/psr-2/#keywords-and-truefalsenull